packages/devtools_app/lib/src/inspector/flutter/inspector_screen_details_tab.dart

```diff
@@ -171,17 +132,7 @@ class _LayoutDetailsTabState extends State<LayoutDetailsTab>
   @override
   Widget build(BuildContext context) {
     super.build(context);
```

Comment:
What is thsis super.build() call for?

Reply:
Good catch, this is autogenerated when I created the stateful widget.

Reply:
actually nevermind, this call is required because of AutomaticKeepAliveClientMixin

Reply:
I'l add comment